e2e/cluster/local/images/k3s/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Install the all-in-one binary so we can copy our run-time images into the image
```

Comment:
Is there a reason you are not using the official Image directly in the docker-compose?

Reply:
Yes, I tried to capture in this comment. I have to recreate a k3s image because the official image declares `/var/lib/rancher` as a VOLUME which means subsequent additions to that area will never show up in an image. If, however, in a Dockerfile you copy content into a location and then declare it a VOLUME afterwards and then run the image with an non-bind mount volume at that location, docker will by default at first-run copy the contents from the image into the volume. This is the trick that I am leveraging.

Reply:
I see, thanks for clarification

Reply:
The other reason I need to re-create a k3s image is because I want the tests to run locally without the need to pull anything which means I need the images tarballs in the k3s image. Doing that via docker-compose within Dapper is problematic because the docker daemon it is using does not have the same filesystem root that the host does which means bind mounts are hopeless.